src/language/parser.js

```diff
@@ -635,3 +634,19 @@ function parseNamedType(parser): NamedType {
     loc: loc(parser, start)
   };
 }
+
+export var ParseOps = {
```

Comment:
Interesting - perhaps all these fn's should get extracted to their own file and exported directly. That file could then support both of these?